lib/ember-app.js

```diff
@@ -86,6 +86,13 @@ EmberApp.prototype.visit = function(path, options) {
     .then(function(instance) {
       return instance.visit(path, bootOptions);
     })
+    .then(function(instance) {
+      var fastbootInfo = instance.lookup('info:-fastboot');
```

Comment:
Don't we already have the FastBootInfo object, since we provide it to the app? Why do we have to look it up again?